tests/lib/snaps/test-snapd-netlink-connector/bin/bind

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+ 
+import socket
+import os
+
+
+if __name__ == "__main__":
+    # Create Netlink socket
+    # For more information about protocols see https://github.com/torvalds/linux/blob/master/include/uapi/linux/netlink.h
+    netlink_connector_protocol = 11
+    sock = socket.socket(socket.AF_NETLINK, socket.SOCK_DGRAM, netlink_connector_protocol)
```

Comment:
I guess this will be cleaned up on exit anyway, but a nicer way of writing this in Python would be:

```
with socket.socket(....) as sock:
    ...
    sock.bind((os.getpid(), 1))
    print('netlink connect binded')
```
And if writing for an older Python 3 (3.2?):

```
from contextlib import closing

with closing(socket.socket(...)) as sock:
    ....
```